topi/python/topi/nn/depthwise_conv2d.py

```diff
@@ -1,13 +1,35 @@
-# pylint: disable=invalid-name, unused-variable, too-many-locals
+# pylint: disable=invalid-name, unused-variable, too-many-locals, unused-argument
 """Depthwise convolution operators"""
 from __future__ import absolute_import as _abs
+from collections import namedtuple
 import tvm
 
 from .dilate import dilate
 from .pad import pad
 from .util import get_pad_tuple
 from ..util import simplify
 
+# workload description of depthwise-conv2d
+Workload = namedtuple('Workload',
```

Comment:
Do we want dilation in here? Or that's for separate PR?

Reply:
Workload here is for getting default schedule, since dilation so far does not impact how we calculate configs, I'd rather keep it simple for now.

Reply:
Makes sense. This is resolved from my side.